lib/resources/postgres_session.rb

```diff
@@ -39,25 +41,32 @@ class PostgresSession < Inspec.resource(1)
         its('output') { should eq('') }
       end
     "
+    attr_reader :user, :pass, :host, :db, :escaped_query, :psql_cmd
```

Comment:
I expose user, pass and db check the constructor. I exposed db, escaped_query and psql_cmd to test that each of the helper functions was working as expected. 

I guess I could just expose user, pass, host and psql_cmd and that would enable us to check that all the parts are working as expected over time.